aptos-move/block-executor/src/view.rs

```diff
@@ -1495,19 +1601,25 @@ impl<'a, T: Transaction, S: TStateView<Key = T::Key>> TResourceGroupView for Lat
     ) -> PartialVMResult<ResourceGroupSize> {
         let mut group_read = match &self.latest_view {
             ViewState::Sync(state) => state.read_group_size(group_key, self.txn_idx)?,
-            ViewState::Unsync(state) => state.unsync_map.get_group_size(group_key),
+            ViewState::Unsync(state) => match state.unsync_map.get_group_size(group_key) {
+                Some(group_size) => GroupReadResult::GroupSize(group_size),
+                None => GroupReadResult::Uninitialized,
+            },
         };
 
         if matches!(group_read, GroupReadResult::Uninitialized) {
             self.initialize_mvhashmap_base_group_contents(group_key)?;
 
             group_read = match &self.latest_view {
                 ViewState::Sync(state) => state.read_group_size(group_key, self.txn_idx)?,
-                ViewState::Unsync(state) => state.unsync_map.get_group_size(group_key),
+                ViewState::Unsync(state) => match state.unsync_map.get_group_size(group_key) {
+                    Some(group_size) => GroupReadResult::GroupSize(group_size),
+                    None => GroupReadResult::Uninitialized,
```

Comment:
This should be unreachable now?

Reply:
returned a PanicError => VMError after unifying the interfaces